src/pages/workspace/accounting/xero/XeroMapTrackingCategoryConfigurationPage.tsx

```diff
@@ -1,61 +1,67 @@
+import {useRoute} from '@react-navigation/native';
 import React, {useCallback, useMemo} from 'react';
 import ConnectionLayout from '@components/ConnectionLayout';
 import SelectionList from '@components/SelectionList';
 import RadioListItem from '@components/SelectionList/RadioListItem';
 import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
 import * as Connections from '@libs/actions/connections';
-import {getTrackingCategory} from '@libs/actions/connections/ConnectToXero';
 import Navigation from '@libs/Navigation/Navigation';
 import type {WithPolicyProps} from '@pages/workspace/withPolicy';
 import withPolicyConnections from '@pages/workspace/withPolicyConnections';
 import CONST from '@src/CONST';
 import type {TranslationPaths} from '@src/languages/types';
 import ROUTES from '@src/ROUTES';
 
-function XeroMapCostCentersToConfigurationPage({policy}: WithPolicyProps) {
+type RouteParams = {
+    categoryId?: string;
+    categoryName?: string;
+};
+
+function XeroMapTrackingCategoryConfigurationPage({policy}: WithPolicyProps) {
     const {translate} = useLocalize();
+    const route = useRoute();
+    const params = route.params as RouteParams;
     const styles = useThemeStyles();
-
+    const categoryId = params?.categoryId ?? '';
+    const categoryName = params?.categoryName ?? '';
     const policyID = policy?.id ?? '';
 
-    const category = getTrackingCategory(policy, CONST.XERO_CONFIG.TRACKING_CATEGORY_FIELDS.COST_CENTERS);
-
     const optionsList = useMemo(
         () =>
             Object.values(CONST.XERO_CONFIG.TRACKING_CATEGORY_OPTIONS).map((option) => ({
                 value: option,
                 text: translate(`workspace.xero.trackingCategoriesOptions.${option.toLowerCase()}` as TranslationPaths),
                 keyForList: option,
-                isSelected: option.toLowerCase() === category?.value?.toLowerCase(),
+                isSelected: option.toLowerCase() === categoryName.toLowerCase(),
```

Comment:
This condition is now outdated right? 
Our `condition` will be `DEFAULT` or `TAG` meanwhile our `categoryName` could be anything?

Reply:
My typo, I mean the `option ` value

Reply:
You're right, we should update code to reflect that. 

I've changed it [here](https://github.com/Expensify/App/pull/42593/commits/3d30445d5d92e01ca88e7fb01802deadd745aabb)